pkg/auth/providers/keycloakoidc/keycloak_client.go

```diff
@@ -170,7 +159,7 @@ func (k *KClient) getFromKeyCloak(accessToken, url string, config *v32.OIDCConfi
 	}
 	req.Header.Add("Authorization", "token "+accessToken)
 	req.Header.Add("Accept", "application/json")
-	resp, err := kHTTPClient.httpClient.Do(req)
+	resp, err := k.httpClient.Do(req)
```

Comment:
We should also call defer resp.Body.Close() in here to ensure the http response is closed